README.md

```diff
@@ -57,9 +57,9 @@ and run it under `$USER`, in this case you will not be able to login as someone
 If you wish to deploy Teleport inside a Docker container:
 
 ```
-# This command will pull the Teleport container image for version 2.7.3
+# This command will pull the Teleport container image for version 3.2.6
 # Replace 2.7.3 with the version you need:
-$ docker pull quay.io/gravitational/teleport:2.7.3
+$ docker pull quay.io/gravitational/teleport:3.2.6
```

Comment:
Does Quay have the option of allowing pulling of the latest image? That would probably be better otherwise we'll haev to upgrade docs for every release.

Reply:
It does, but we need to tag a latest version. 


```➜  ~ docker pull quay.io/gravitational/teleport
Using default tag: latest
Error response from daemon: manifest for quay.io/gravitational/teleport:latest not found
```